tests/system/CommonFunctionsTest.php

```diff
@@ -390,9 +390,9 @@ public function testReallyWritable()
 
     public function testSlashItem()
     {
-        $this->assertSame('/', slash_item('cookiePath')); // slash already there
-        $this->assertSame('', slash_item('cookieDomain')); // empty, so untouched
-        $this->assertSame('en/', slash_item('defaultLocale')); // slash appended
+        $this->assertSame('/', slash_item('cookiePath'));
+        $this->assertNull(slash_item('cookieDomain'));
```

Comment:
`$cookieDomain` is empty string, not null.
https://github.com/codeigniter4/CodeIgniter4/blob/451ebb0e4c9fab01312cf1091bd24cf741708825/app/Config/App.php#L261

```suggestion
        $this->assertSame('', slash_item('cookieDomain'));
```

Reply:
And a test for `slash_item('item_does_not_exist')` is needed.